source/game.gamescriptexpression.bmx

```diff
@@ -215,14 +221,13 @@ Function SEFN_programmelicence:SToken(params:STokenGroup Var, context:SScriptExp
 		Case "broadcastlimit"          Return New SToken( TK_NUMBER, licence.GetBroadcastLimit(), params.GetToken(0) )
 		Case "hasbroadcastlimit"       Return New SToken( TK_BOOLEAN, licence.HasBroadcastLimit(), params.GetToken(0) )
 		Case "episodenumber"           Return New SToken( TK_NUMBER, licence.GetEpisodeNumber(), params.GetToken(0) )
```

Comment:
Episodenumber vlt. noch mit nach oben? Gleiches fuer "country"?
Die ID/GUID wird wohl eher seltener abgerufen als andere Informationen. "country" koennte theoretisch genommen werden um den Personengenerator zu fuettern ("XYZ in einer Gastrolle") - auch wenn dort das Produktionsland einfach immer "Senderkarte-Land" sein wuerde und somit "diese" Information genutzt werden koennte. In dem Fall kann "country" auch unten bleiben.